.circleci/config.yml

```diff
@@ -109,6 +109,16 @@ commands:
     steps:
       - run: echo "${DOCKER_PASS}" | docker login --username ${DOCKER_USER} --password-stdin
 
+  install-go:
```

Comment:
question: why not use our golang CI image?

Reply:
I tried, but I need docker to run postgres and get some crazy errors trying to start docker in the `cimg:golang` image:
```
circleci@fab86abe2cd6:~/project$ sudo service docker start
 * Starting Docker: docker                                                                                                                                                                                                                                                                                                                                    [ OK ] 
circleci@fab86abe2cd6:~/project$ tail -f /var/log/docker.log 
time="2020-10-19T15:36:44.770568800Z" level=warning msg="Unable to setup quota: operation not permitted\n"
time="2020-10-19T15:36:44.785362400Z" level=info msg="Loading containers: start."
time="2020-10-19T15:36:44.785543900Z" level=warning msg="Running modprobe bridge br_netfilter failed with message: , error: exec: \"modprobe\": executable file not found in $PATH"
time="2020-10-19T15:36:44.787341400Z" level=warning msg="Running iptables --wait -t nat -L -n failed with message: `iptables v1.6.1: can't initialize iptables table `nat': Permission denied (you must be root)\nPerhaps iptables or your kernel needs to be upgraded.`, error: exit status 3"
time="2020-10-19T15:36:44.809573900Z" level=info msg="stopping event stream following graceful shutdown" error="<nil>" module=libcontainerd namespace=moby
time="2020-10-19T15:36:44.810338600Z" level=info msg="stopping event stream following graceful shutdown" error="context canceled" module=libcontainerd namespace=plugins.moby
time="2020-10-19T15:36:44.810368900Z" level=info msg="stopping healthcheck following graceful shutdown" module=libcontainerd
failed to start daemon: Error initializing network controller: error obtaining controller instance: failed to create NAT chain DOCKER: iptables failed: iptables -t nat -N DOCKER: iptables v1.6.1: can't initialize iptables table `nat': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
 (exit status 3)
```

Reply:
Another fun alternative: you could run postgres manually (without docker) in some random local directory:

```shell
mkdir sock data
initdb -D $(pwd)/data
postgres --unix-socket-directories=$(pwd)/sock -D $(pwd)/data
```

Which you can then connect to with `psql --host $(pwd)/sock postgres`

Reply:
I did this about that, but when we support elastic I'm going to need to run it to run the elastic filtering tests I plan to write, so docker seems like the way to go to save some pain there.